src/app/routes/receive-routes.tsx

```diff
@@ -0,0 +1,27 @@
+import { Route } from 'react-router-dom';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { ReceiveBtcModal } from '@app/pages/receive/receive-btc';
+import { ReceiveModal } from '@app/pages/receive/receive-modal';
+import { ReceiveOrdinalModal } from '@app/pages/receive/receive-ordinal';
+import { ReceiveStxModal } from '@app/pages/receive/receive-stx';
+
```

Comment:
I split `app-routes` up as it was difficult to work with. 

I could refactor further to change this to object config:

```
export const receiveRoutesArray = [
  {
    path: RouteUrls.Receive,
    element: <ReceiveModal />,
    routes: [
      {
        path: RouteUrls.ReceiveStx,
        element: <ReceiveStxModal />,
      },

```